common/config/src/properties.rs

```diff
@@ -172,6 +173,13 @@ fn from_property_map(map: HashMap<String, String>) -> Result<Config, ConfigError
         result.http.retry_dir = Some(PathBuf::from(value));
     }
 
+    if let Some(value) = map.get(&RETRY_DISK_LIMIT) {
+        let limit = u64::from_str(value).map_err(|e| {
```

Comment:
What about using something like https://crates.io/crates/humansize to allow users to put in friendly strings like "1GB" or "200MB" etc?

Reply:
humansize seems to only convert int values into human readable strings. https://crates.io/crates/humanize-rs looks like it's the popular choice for parsing strings back into ints. I'm going to plan on going with that .

Reply:
Commit https://github.com/logdna/logdna-agent-v2/pull/279/commits/8965a89b0ecfa3ac842b3db4e1d5eb5426b0d4f6 adds humanize_rs as a parser for command line argument and environment values. Getting that same logic to work with serde and the YAML format has been more challenging and isn't yet done in the PR.

Reply:
I updated the parsing logic for the yaml file so it also understand a humanized string in commit https://github.com/logdna/logdna-agent-v2/pull/279/commits/0535082caaa4326836d35c2a9120bd7fd1be8ee8.